projects/storefrontapp-e2e-cypress/cypress/e2e/accessibility/account-settings-close-account.a11y-e2e.cy.ts

```diff
@@ -0,0 +1,32 @@
+/*
+ * SPDX-FileCopyrightText: 2025 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { standardUser } from '../../sample-data/shared-users';
+
+const CLOSE_ACCOUNT_URL = '/my-account/close-account';
+
+/**
+ * This test checks accessibility concerns on the Account Settings Close Account page using Access Continuum
+ */
+describe('Account Settings / Close Account Page Accessibility', () => {
+  before(() => {
+    cy.a11yContinuumSetup();
+  });
+
+  it('initial page load', () => {
+    cy.requireLoggedIn(standardUser);
+    cy.visit(CLOSE_ACCOUNT_URL);
+
+    cy.get('main').a11yRunContinuumTest();
+  });
+
+  it('confirm Account Closure Modal', () => {
```

Comment:
I think we could make this test run faster by disabling test isolation so that we don't need to reload pages..

Reply:
This is relevant for the other test suites too.